modules/custom/d_geysir/d_geysir.module

```diff
@@ -21,7 +21,7 @@ function d_geysir_form_alter(&$form, FormStateInterface $form_state, $form_id) {
       foreach ($form['description'] as $paragraph_name => $attributes) {
         $file_server_path = $root_path . '/' . $module_path . '/images/' . $paragraph_name . '.png';
         if (file_exists($file_server_path)) {
-          $file_url = $base_url . $module_path . '/images/' . $paragraph_name . '.png';
+          $file_url = $base_url . '/' . $module_path . '/images/' . $paragraph_name . '.png';
```

Comment:
I wonder if there is a case that this could break those url, maybe it would be just safer to add there an `if` before foreach and check there if  `$base_url` isn't empty - in that case I would rather add there a slash like this `$base .= '/'`

@grzegorz-pietrzak-droptica WDYT ? 
When I was doing some tests today I was able to create a configuration in which this fix :point_up:  was breaking the icons :thinking:

Reply:
As @rafal-sereda-droptica wrote we need to add `\` only when `$base_url` isn't empty.